administrator/components/com_workflow/tmpl/states/default.php

```diff
@@ -27,24 +38,94 @@
 		<div class="col-md-10">
 			<div id="j-main-container" class="j-main-container">
 				<?php
-					// Search tools bar
-					echo \JLayoutHelper::render('joomla.searchtools.default', array('view' => $this));
+				// Search tools bar
+				echo \JLayoutHelper::render('joomla.searchtools.default', array('view' => $this));
 				?>
 				<?php if (empty($this->states)) : ?>
 					<div class="alert alert-warning alert-no-items">
 						<?php echo JText::_('JGLOBAL_NO_MATCHING_RESULTS'); ?>
 					</div>
 				<?php else: ?>
-					<table class="table table-striped" id="emailList">
-						<thead><?php echo $this->loadTemplate('head');?></thead>
+					<table class="table table-striped">
+						<thead>
+							<tr>
+								<th style="width:1%" class="nowrap text-center hidden-sm-down">
+									<?php echo JHtml::_('searchtools.sort', '', 's.ordering', $listDirn, $listOrder, null, 'asc', 'JGRID_HEADING_ORDERING', 'icon-menu-2'); ?>
+								</th>
+								<th style="width:1%" class="nowrap text-center hidden-sm-down">
+									<?php echo JHtml::_('grid.checkall'); ?>
+								</th>
+								<th style="width:1%" class="nowrap text-center hidden-sm-down">
+									<?php echo JHtml::_('searchtools.sort', 'JSTATUS', 's.condition', $listDirn, $listOrder); ?>
+								</th>
+								<th style="width:1%" class="text-center nowrap hidden-sm-down">
+									<?php echo JText::_('COM_WORKFLOW_DEFAULT'); ?>
+								</th>
+								<th style="width:10%" class="nowrap hidden-sm-down">
+									<?php echo JHtml::_('searchtools.sort', 'COM_WORKFLOW_TITLE', 's.title', $listDirn, $listOrder); ?>
+								</th>
+								<th style="width:10%" class="nowrap text-center hidden-sm-down">
+									<?php echo JHtml::_('searchtools.sort', 'COM_WORKFLOW_CONDITION', 's.condition', $listDirn, $listOrder); ?>
+								</th>
+								<th style="width:10%" class="nowrap text-right hidden-sm-down">
+									<?php echo JHtml::_('searchtools.sort', 'COM_WORKFLOW_ID', 's.id', $listDirn, $listOrder); ?>
+								</th>
+							</tr>
+						</thead>
+						<tbody class="js-draggable" data-url="<?php echo $saveOrderingUrl; ?>" data-direction="<?php echo strtolower($listDirn); ?>">
+							<?php foreach ($this->states as $i => $item):
+								$link = JRoute::_('index.php?option=com_workflow&task=state.edit&id=' . $item->id . '&workflow_id=' . $workflowID . '&extension=' . $this->extension);
+
+								$canChange  = $user->authorise('core.edit.state', 'com_workflow.state.' . $item->id);
+								?>
+								<tr class="row<?php echo $i % 2; ?>">
```

Comment:
Here you have to define the grouping parameter (=> see workflow default.php)

Reply:
ok